### ISSUE TITLE
File becomes a project in the summary and is empty

### ISSUE THREAD

**Issue by maaaathis** [muety/wakapi#625]
**Describe the bug**
In the files summary it is possible to click on a specific file. As a user, I would now expect to find further information about my activity in the file on the page that then opens. However, this is not the case and the summary is opened for the file as a project. Here, of course, everything is empty because there is no data about a project with the exact file path of my file. The heading of the summary page then looks something like this: 'Project "/Users/[...]/WebstormProjects/[...]/src/[...]"'

TL;DR: Clicking on a single file in the file summary opens a new summary page, but the project is the file (and a file is not a project, which is why it is always empty).

**System information**
Wakapi is on the newest version: v2.10.5 @ sqlite3

**Comment by muety** [comment]
Thanks for reporting this1 There is no such thing like a "file summary", so files shouldn't be clickable. Will fix this.
